
float32 TargetYaw
float32 DroneYaw
float32 YawVel
float32 PitchVel
float32 RollVel

float32 DroneX
float32 TargetX
float32 FrameDroneX
float32 FrameTargetX

float32 DroneY
float32 TargetY
float32 FrameDroneY
float32 FrameTargetY

float32 DroneAltitude
float32 TargetAltitude

string mode



